.github/workflows/build_linux.yaml

```diff
@@ -94,16 +94,8 @@ jobs:
 
             - name: Check ldd
               run: |
-                  ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node
-                  if [ "$SQLCIPHER_STATIC" == "1" ]; then
-                      ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node | grep -v libsqlcipher.so.0
-                      ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node | grep libcrypto.so.1.1
-                  else
-                      ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node | grep libsqlcipher.so.0
-                      ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node | grep -v libcrypto.so.1.1
-                  fi
-              env:
-                  SQLCIPHER_STATIC: ${{ inputs.sqlcipher == 'static' && '1' || '' }}
+                  ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node | grep -v libsqlcipher.so.0
+                  ldd dist/linux-unpacked/resources/app.asar.unpacked/node_modules/matrix-seshat/native/index.node | grep -v libcrypto.so.1.1
```

Comment:
This check doesn't actually do what we want (its wrong in the first place) - https://github.com/vector-im/element-desktop/pull/446/files#diff-d137c42776727a733a3d1879a8096fe7ecb4c6962b3a0cce48984c99ba110e73R153-R164 fixes it